enhancements/etcd/automated-backups-no-config.md

```diff
@@ -0,0 +1,256 @@
+  - This could be a future enhancement or extension to the API.
+  - Having backups in an independent storage from OCP cluster is crucial for Disaster Recovery scenarios.
+* Automate cluster restoration.
+* Provide automated backups for non-self hosted architectures like Hypershift.
+
+## Proposal
+
+To enable automated etcd backups of an Openshift cluster, a default backup option is to be taking sing a `SideCar` container within each etcd pod, with no configuration from user.
+
+### User Stories
+
+* As a cluster administrator I would like OCP cluster backup to be taken without configuration.
+* As a cluster administrator I would like to schedule recurring OCP cluster backups so that I have a recent cluster state to recover from in the event of quorum loss (i.e. losing a majority of control-plane nodes).
+
+### API Extensions
+
+No [API](https://github.com/openshift/enhancements/blob/master/enhancements/etcd/automated-backups.md#api-extensions) changes are required, since this approach work independently with default config.
+
+### Topology Considerations
+TBD
+#### Hypershift / Hosted Control Planes
+TBD
+#### Standalone Clusters
+TBD
+#### Single-node Deployments or MicroShift
+TBD
+### Workflow Description
+TBD
+
+## Design Details
+
+A `SideCar` container is being added within each etcd pod in Openshift cluster in order to take backups without any provided configuration from user's perspective.
+
+As the `SideCar` is being deployed alongside each etcd cluster member, it is possible to keep backups across all master nodes.
+
+Since this work will be enabled with no configuration, it is possible to use define a default values for the `Schedule`, `Retention` independently.
+
+This work utilizes `hostPath` approach, since the `SideCar` container has access to the etcd pod's file system.
+
+Before deciding upon this approach, several approaches that have been investigated. See [Alternatives](#Alternatives).
+
+### Open Questions
+
+* How to distinguish between `NoConfig` backups and configs that are triggered using `EtcdBackup` CR.
+  * The `NoConfig` backups relies on `EtcdBackup` CR with `default` name.
+  * The cluster-etcd-operator reacts to the `default` CR by deploying the backup sidecar containers alongside each etcd member. 
+
+
+* Upon enabling the `AutomatedBackup` feature gate, which approach should be used and according to what criteria.
+  * As the `Noconfig` backups is orthogonal to the [automated backup of etcd](https://docs.openshift.com/container-platform/4.15/backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.html#creating-automated-etcd-backups_backup-etcd), it has been decided to use the same feature gate.
+  
+
+* How should the current backup controller distinguish between both approaches.
+  * The `NoConfig` backups relies on `EtcdBackup` CR with `default` name.
+  
+
+* How to disable the `NoConfig` behaviour, if a user want to.
+  * This has not been decided or implemented yet. 
```

Comment:
If the installer created the object, and it was unmanaged, the user would just delete it